### ISSUE TITLE
Index populating failed for populator "Umbraco.Cms.Infrastructure.Examine.ContentIndexPopulator"

### ISSUE THREAD

**Issue by chriskarkowsky** [umbraco/Umbraco-CMS#14910]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

12.0.1

### Bug summary

Error in the log viewer is presenting as: Index populating failed for populator "Umbraco.Cms.Infrastructure.Examine.ContentIndexPopulator"

Once this issue is logged, we are then unable to Save any items in the backoffice.

### Steps to reproduce

We deploy or Umbraco app to Azure Web app.  

On boot, I save a Design file, which is successful.

Then immediately in the, log it produces this:

<img width="985" alt="Screenshot 2023-10-03 at 4 17 31 PM" src="https://github.com/umbraco/Umbraco-CMS/assets/24321869/518ddd45-4fae-4590-a54e-8bd934ac551d">

I then try to save the file again, and get:

<img width="605" alt="Screenshot 2023-10-03 at 4 20 38 PM" src="https://github.com/umbraco/Umbraco-CMS/assets/24321869/a0c8fa9e-6a32-4136-87e9-89be7a488e92">

So I am unable to save anything now.

The Details of the errors:
`
System.InvalidOperationException: The Scope 3653e316-1f1f-488f-98e2-dd246cbd1a1b being disposed is not the Ambient Scope 92fa41bc-7b3b-465d-92d7-8058e168425b. This typically indicates that a child Scope was not disposed, or flowed to a child thread that was not awaited, or concurrent threads are accessing the same Scope (Ambient context) which is not supported. If using Task.Run (or similar) as a fire and forget tasks or to run threads in parallel you must suppress execution context flow with ExecutionContext.SuppressFlow() and ExecutionContext.RestoreFlow().
   at Umbraco.Cms.Infrastructure.Scoping.Scope.Dispose()
   at Umbraco.Cms.Core.Services.ContentTypeServiceBase`2.Get(Guid id)
   at Umbraco.Cms.Core.PropertyEditors.NestedPropertyIndexValueFactoryBase`2.Handle(TSerialized deserializedPropertyValue, IProperty property, String culture, String segment, Boolean published, IEnumerable`1 availableCultures)
   at Umbraco.Cms.Core.PropertyEditors.JsonPropertyIndexValueFactoryBase`1.GetIndexValues(IProperty property, String culture, String segment, Boolean published, IEnumerable`1 availableCultures)
   at Umbraco.Cms.Core.PropertyEditors.NestedPropertyIndexValueFactoryBase`2.GetNestedResults(String keyPrefix, String culture, String segment, Boolean published, IDictionary`2 propertyTypeDictionary, TItem nestedContentRowValue, IEnumerable`1 availableCultures)+MoveNext()
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at Umbraco.Cms.Core.PropertyEditors.NestedPropertyIndexValueFactoryBase`2.Handle(TSerialized deserializedPropertyValue, IProperty property, String culture, String segment, Boolean published, IEnumerable`1 availableCultures)
   at Umbraco.Cms.Core.PropertyEditors.JsonPropertyIndexValueFactoryBase`1.GetIndexValues(IProperty property, String culture, String segment, Boolean published, IEnumerable`1 availableCultures)
   at Umbraco.Cms.Infrastructure.Examine.BaseValueSetBuilder`1.AddPropertyValue(IProperty property, String culture, String segment, IDictionary`2 values, IEnumerable`1 availableCultures)
   at Umbraco.Cms.Infrastructure.Examine.ContentValueSetBuilder.GetValueSetsEnumerable(IContent[] content, Dictionary`2 creatorIds, Dictionary`2 writerIds)+MoveNext()
   at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)
   at System.Collections.Generic.EnumerableHelpers.ToArray[T](IEnumerable`1 source)
   at Umbraco.Cms.Infrastructure.Examine.ContentIndexPopulator.IndexAllContent(Int32 contentParentId, Int32 pageIndex, Int32 pageSize, IReadOnlyList`1 indexes)
   at Umbraco.Cms.Infrastructure.Examine.ContentIndexPopulator.PopulateIndexes(IReadOnlyList`1 indexes)
   at Umbraco.Cms.Infrastructure.Examine.IndexPopulator.Populate(IIndex[] indexes)
   at Umbraco.Cms.Infrastructure.Examine.ExamineIndexRebuilder.RebuildIndexes(Boolean onlyEmptyIndexes, TimeSpan delay, CancellationToken cancellationToken)
`

`
System.InvalidOperationException: The Scope 92fa41bc-7b3b-465d-92d7-8058e168425b being disposed is not the Ambient Scope ed835d65-c53d-42ec-a2b8-6da2c2caa822. This typically indicates that a child Scope was not disposed, or flowed to a child thread that was not awaited, or concurrent threads are accessing the same Scope (Ambient context) which is not supported. If using Task.Run (or similar) as a fire and forget tasks or to run threads in parallel you must suppress execution context flow with ExecutionContext.SuppressFlow() and ExecutionContext.RestoreFlow().
   at Umbraco.Cms.Infrastructure.Scoping.Scope.Dispose()
   at Umbraco.Cms.Infrastructure.Examine.ExamineUmbracoIndexingHandler.DeferredReIndexForContent.<>c__DisplayClass6_0.<Execute>b__0(CancellationToken cancellationToken)
   at Umbraco.Cms.Infrastructure.HostedServices.QueuedHostedService.BackgroundProcessing(CancellationToken stoppingToken)
`

### Expected result / actual result

Reindexing of the page should be successful and not lock the database to prevent further editing.

**Comment by elit0451** [comment]
Seems like a duplicate of #14195. I will close this one so we can keep the discussion in one place
